src/Shapefile.jl

```diff
@@ -123,6 +271,22 @@ const SHAPETYPE = Dict{Int32,DataType}(
     31 => MultiPatch,
 )
 
+"""
+    Handle
+
+    Handle(path::AbstractString, [indexpath::AbstractString])
+
+Why is it called handle again?
```

Comment:
Maybe because it's an open file handle?

Reply:
Yeah I guess thats the original reason, but it isn't really a handle, because it just loads the whole file eagerly to memory. It doesn't even keep the string name anywhere.